tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -150,6 +157,26 @@ public void SetupGet_can_setup_readwrite_property()
 			Assert.Equal(42, actual);
 		}
 
+		[Fact]
+		public void SetUpGet_can_automock()
+		{
+			this.protectedMock.SetupGet(m => m.Nested.Value).Returns(42);
+
+			var actual = mock.Object.GetNested().Value;
+
+			Assert.Equal(42, actual);
+		}
+
+		[Fact]
+		public void SetupGet_can_setup_virtual_property()
+		{
+			this.protectedMock.SetupGet(m => m.VirtualGet).Returns(42);
+
+			var actual = mock.Object.GetVirtual();
+
+			Assert.Equal(42, actual);
+		}
+
```

Comment:
&dagger;) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.